src/views/TabView/TabBarTop.js

```diff
@@ -78,10 +78,17 @@ export default class TabBarTop extends PureComponent<DefaultProps, Props, void>
       outputRange,
     });
     const label = this.props.getLabelText(scene);
+    if (typeof label === 'string') {
+      return (
+        <Animated.Text style={[styles.label, labelStyle, { color }]}>
+          {label}
+        </Animated.Text>
+      );
+    }
     return (
```

Comment:
I think this can just be `return label`. `labelStyle` is supposed to be text style